google_shopping_functions.js

```diff
@@ -9,7 +9,63 @@ var getItemsCount = function(itemData){
 //DEFINE FUNCTIONS HERE
 //
 
+/*Part 1 - Create a function called getItems that simply returns the items array from the google product object.*/
 
+var getItems = function(objectData){
+
+	return objectData.items;
+}
+
+/*Part 2 - Create a function called getItemsByBrand that takes an item array returns a new array of all items of a specified brand*/
+
+var itemsByBrand = [];
+var getItemsByBrand = function(items, thebrand){
+var q = 0;
```

Comment:
Nice job using a counter to add to your items array. An easier (and more conventional) way to do this though would be to use the array.push() method.

```
if(items[i].product.brand.toLowerCase() === thebrand){
    itemsByBrand.push(items[i]);
}
```